pyepal/plotting/__init__.py

```diff
@@ -166,6 +166,63 @@ def plot_histogram(y: np.ndarray, palinstance, ax):  # pylint:disable=invalid-na
     )
 
 
+def plot_res(  # pylint:disable=invalid-name
+    y: np.array,
+    palinstance,
```

Comment:
also not important for this PR, but we might want to do this in a future enhancement: We should also annotate the other types in the functions

Reply:
sounds good